migrations/2_dev_dependencies.js

```diff
@@ -1,4 +1,4 @@
-const migrateBatchExchange = require("@gnosis.pm/dex-contracts/src/migration/PoC_dfusion")
+const migrateBatchExchange = require("@gnosis.pm/dex-contracts/migration/PoC_dfusion")
```

Comment:
I think this one should stay as is given that `PoC_dfusion` is a JS file that isn't re-exported.

Reply:
```suggestion
const migrateBatchExchange = require("@gnosis.pm/dex-contracts/src/migration/PoC_dfusion")
```